CHANGES/domain-enablement.feature

```diff
@@ -0,0 +1,2 @@
+Add partial support for Domains. The plugin can be installed with the feature turned on, but only
+the default domain will properly work.
```

Comment:
I would much prefer if we can say "it only functions in the default domain". By that i mean, that you could not create a container repository outside of the default domain for example.